demo.html

```diff
@@ -1,52 +1,86 @@
 <!DOCTYPE html>
 <html>
-<head>
-
-  <title>Parallax Demo</title>
-
-  <!-- Parallax implementation -->
-  <script src="prototype/implementation.js"></script>
-
-  <!-- Code highlighting -->
-  <link rel="stylesheet"
-        href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css">
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
-  <style>
-  html, body {
-    margin: 0;
-  }
-
-  .header {
-    background: url("images/sunset.jpg");
-    background-size: cover;
-    height: 30vh;
-  }
-
-  .content {
-    position: relative;
-    background: white;
-    padding: 2em;
-  }
-  </style>
-</head>
-<body>
-
-  <parallax-container>
-    <parallax-content rate="0.3">
-      <div class="header"></div>
-    </parallax-content>
-    <div class="content">
-      <h1>Parallax</h1>
-
-      <p>Parallax is a common technique used in application interfaces to make the interface appear to have real depth.</p>
-
-      <p>This demo shows the use of a polyfill of this feature using custom elements. The header is moving at a rate of 30% of the scroll. Try changing it and seeing the rate update.</p>
-
-      <h2>How to use it</h2>
-
-      <p>To add an element which moves with parallax, put a <code>&lt;parallax-content&gt;</code> element within a <code>&lt;parallax-container&gt;</code> element. The <code>rate</code> attribute on the <code>&lt;parallax-content&gt;</code> element defines how quickly the item moves with respect to the scroll movement.</p>
-
-      <pre><code class="html"><!--
+	<head>
```

Comment:
Can you remove the changed indent? Generally speaking the style reflects https://google.github.io/styleguide/htmlcssguide.html where a 2-space indent is used.

Reply:
Sorry about that. Forgot to change the indent size in vscode.

Reply:
Fixed in 2efe3f6.